zulipterminal/model.py

```diff
@@ -300,7 +300,7 @@ def get_messages(self, *,
             if first_anchor and response['anchor'] != 10000000000000000:
                 self.index['pointer'][str(self.narrow)] = response['anchor']
             query_range = num_after + num_before + 1
-            if len(response['messages']) < (query_range):
+            if len(response['messages']) <= (query_range):
```

Comment:
You could also get rid of the unnecessary parens around `query_range` while you are at it.

Reply:
Sure. Thanks for pointing it out! :)